openreview/tools.py

```diff
@@ -1380,6 +1408,7 @@ def get_neurips_profile_info(profile, n_years=3):
             domains.update(openreview.tools.subdomains(email))
 
     ## Publications section: get publications within last n years
+    curr_year = datetime.datetime.now().year
```

Comment:
what do you need to re-set curr_year? I see the filter is using `>` instead of `>=`:

```
        if year > cut_off_year:
            publications.add(pub.id)
```

Reply:
What do you mean re-set? This is the first that the variable is declared.

> I see the filter is using > instead of >=

What filter are you talking about?

Reply:
sorry, I misread the variable, I thought it was "cut_off_year"